### DISCUSSION TITLE
Please update this repository

### DISCUSSION THREAD

**Discussion by shreyanshhub** [bsc-iitm/smac#2] [Show and tell]
Hii,
I have made 12-13 more commits on my repository (https://github.com/shreyanshhub/math_and_stats_society) where I have added few more pages and some CSS files too along with images , please update this repository with the same or will you please grant me access to this repository as in near future I have to keep this smac website updated.
Thank you

**Comment by Siddhesh-Agarwal** [comment]
Can I go ahead and migrate this repository from HTML to React.js?

**Reply by shreyanshhub** [reply]
No
